assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -3019,6 +3021,16 @@ protected String build_instructor_new_edit_assignment_context(VelocityPortlet po
                 }
                 context.put("groupsWithUserSubmission", groupsWithUserSubmission);
             }
+        }else if (isAssignmentsByRelatedGroups){
+            try{
+                Site site = siteService.getSite(siteId);
```

Comment:
Above you can use existing code, look at this:

https://github.com/sakaiproject/sakai/blob/eb4036434ba609481c75209da5217162a0628877/assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java#L2975

Reply:
I try using this site definition, but is inside a try catch, so i try also getting the `Site site = null` before the try catch but get some error